src/pymodaq_gui/parameter/utils.py

```diff
@@ -166,14 +166,33 @@ def iter_children(param, childlist=[]):
     return childlist
 
 
-def iter_children_params(param, childlist=[]):
+def iter_children_params(param, childlist=[], filter_type=[], filter_name=[], selectFilter=False):
```

Comment:
use tuples that are immutable for your filters, and use lower case for arguments